Libraries/src/Amazon.Lambda.AmazonMQEvents/ActiveMQEvent.cs

```diff
@@ -0,0 +1,94 @@
+﻿namespace Amazon.Lambda.AmazonMQEvents
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Apache ActiveMQ event
+    /// https://docs.aws.amazon.com/lambda/latest/dg/with-mq.html
+    /// </summary>
+    public class ActiveMQEvent
+    {
+        /// <summary>
+        /// The source of the event.
+        /// </summary>
+        public string EventSource { get; set; }
+
+        /// <summary>
+        /// The ARN of the event source.
+        /// </summary>
+        public string EventSourceArn { get; set; }
+
+        /// <summary>
+        /// List of ActiveMQ messages.
+        /// </summary>
+        public IList<ActiveMQMessage> Messages { get; set; }
+
+        /// <summary>
+        /// Kafka event record.
+        /// </summary>
+        public class ActiveMQMessage
+        {
```

Comment:
Missed?
- deliveryMode
- correlationID
- replyTo
- type
- expiration
- priority

FYI 
https://github.com/aws/aws-lambda-java-libs/blob/311f97cc47b414633c62ee96d9df3e8404e14d69/aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/ActiveMQEvent.java#L48

Reply:
As I figured it out earlier, the above Java library appears to be out of date. Please refer https://docs.aws.amazon.com/lambda/latest/dg/with-mq.html for the event structure which was used as reference to develop this POCO class.

Reply:
If Java library is not up to date then could you cut a ticket to Java team to update their package?

Reply:
I have created internal ticket for AWS SDK for Java.